lib/board.rb

```diff
@@ -1,4 +1,7 @@
+require "./lib/renderables"
```

Comment:
I think that we can take out this because the renderable module is getting called through the cell